stock_orderpoint_uom/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import Warning as UserError
```

Comment:
Yes, I've seen It is also mentioned in CONTRIBUTING.md file.

cc @jbeficent should we add this change?